test/parallel/test-crypto-random.js

```diff
@@ -456,12 +456,12 @@ assert.throws(
     }
   );
 
-  crypto.randomInt(0, common.mustCall());
-  crypto.randomInt(0, 0, common.mustCall());
```

Comment:
Shouldn't we keep this test, whether it's supposed to fail or not because of the different set of inputs?

Reply:
Do you mean testing `assert.throws(() => randomInt(0, 0)...)` in addition to the existing `assert.throws(() => crypto.randomInt(0)...)`?

Reply:
Yes.